src/main/java/com/crowdin/client/core/http/exceptions/HttpBulkBadRequestException.java

```diff
@@ -0,0 +1,42 @@
+package com.crowdin.client.core.http.exceptions;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class HttpBulkBadRequestException extends CrowdinApiException{
```

Comment:
Crowdin uses `batch` terminology instead of `bulk`. Please rename all the occurrences to fit the API terminology better

Reply:
fixed